### PR TITLE
Update get_scenario and list_scenario to return serialized RankedProjectArea instead of ProjectAreas

### PR DESCRIPTION
**Pull request by elsiel23**

### CODE REVIEW

File: src/planscape/plan/tests.py [hunk 1]
```diff
@@ -1426,11 +1428,12 @@ def test_get_scenario_ok(self):
         self.assertEqual(scenario['notes'], 'my note')
         self.assertEqual(scenario['priorities'], {
                          'cond1': 2, 'cond2': 3})
-        self.assertEqual(scenario['project_areas'][str(
-            self.project_area.pk)]['geometry'], self.geometry)
-        self.assertEqual(scenario['project_areas'][str(
-            self.project_area.pk)]['properties']['estimated_area_treated'], 200)
         self.assertEqual(scenario['config']['max_budget'], 100)
+        scenario_output = scenario['ranked_project_areas'][str(
```
Comment (strangeties) [inline src/planscape/plan/tests.py:1432]: What does the output look like when there are 2 project areas?
Can "rank" and "weighted_score" be present, too?

File: src/planscape/plan/views.py [hunk 2]
```diff
@@ -715,7 +720,7 @@ def list_scenarios_for_plan(request: HttpRequest) -> HttpResponse:
             [_serialize_scenario(scenario,
                                  weights=ScenarioWeightedPriority.objects.filter(
                                      scenario=scenario),
-                                 areas=ProjectArea.objects.filter(project=scenario.project.pk), project=scenario.project)
+                                 ranked_project_areas=RankedProjectArea.objects.select_related('project_area').filter(scenario=scenario), project=scenario.project)
```
Comment (strangeties) [inline src/planscape/plan/views.py:723]: nit: can this be split into multiple lines?

**Comment by elsiel23** [comment]
@lattemilk FYI!

File: src/planscape/plan/tests.py (hunk 1 above)
Comment (elsiel23) [reply src/planscape/plan/tests.py]: done!

**Comment by runforyourlives** [comment]
This is obsoleted by our current data model and scenario-related APIs.

**Pull request closed without merging**